webapp/pages/profile/_id/_slug.vue

```diff
@@ -393,6 +422,16 @@ export default {
     userName() {
       const { name } = this.user || {}
       return name || this.$t('profile.userAnonym')
+    },
+     tabActivity(str, e) {
+      console.log("tabActivity");
+      console.log(str);
+      console.log(e);
+     // document.removeClass('ds-tab-nav-item-active');
+     // this.class('.ds-tab-nav-item-active')
+     // $$('.ds-tab-nav-item-active').removeClass('ds-tab-nav-item-active');
+     // $$(activTab).class("")
+     // $$(this).attr('class', 'ds-tab-nav-item-active');
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-26T10:45:39Z" title="Sunday, May 26th 2019, 12:45:39 pm +02:00">May 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -393,6 +422,16 @@ export default {
     userName() {
       const { name } = this.user || {}
       return name || this.$t('profile.userAnonym')
+    },
+     tabActivity(str, e) {
+      console.log("tabActivity");
+      console.log(str);
+      console.log(e);
+     // document.removeClass('ds-tab-nav-item-active');
+     // this.class('.ds-tab-nav-item-active')
+     // $$('.ds-tab-nav-item-active').removeClass('ds-tab-nav-item-active');
+     // $$(activTab).class("")
+     // $$(this).attr('class', 'ds-tab-nav-item-active');
```


@ogerly this method does not do anything except `console.log`, please remove.